### DISCUSSION TITLE
Run it in a browser?

### DISCUSSION THREAD

**Discussion by GildedHonour** [Stellarium/stellarium#3122] [General]
Can sky be run in a browser somehow? If so, how?

**Comment by gzotti** [comment]
The sky is outside. 
Stellarium (downloadable desktop version which is developed here) cannot be run in a browser.
Stellarium-Web ( https://stellarium-web.org ) looks similar, but is technically a very different piece of software.

**Comment by GildedHonour** [comment]
stellarium-web.org then

Can it then be used on a web page? Or embedded into one somehow?

**Reply by gzotti** [reply]
In case the link is too deeply hidden on the front page: This is the About box of this program
![image](https://user-images.githubusercontent.com/18099873/227369800-254627cc-6889-4c2f-9bcb-914faa41819b.png)

If you actually want to ask whether you can use the software and embed it into your own website, the About box provides the correct contact information.

**Comment by GildedHonour** [comment]
> Stellarium Web Engine is a JavaScript planetarium renderer using WebGL that can be embedded into a website.

So it can be embedded into html page, right?

Could it be embedded in such a way it'd be a background of a page? On top I'd place some links and other stuff.

**Reply by eternalstarn** [reply]
Hey, did you get an answer to this?

**Reply by gzotti** [reply]
Wrong forum!

**Comment by GildedHonour** [comment]
@gzotti up
